packages/components/src/components/install-guide/record-instructions/IntelliJ.vue

```diff
@@ -73,8 +92,11 @@
         for more information.
       </p>
     </section>
-    <section>
-      <h3>Runnable recording</h3>
+    <br />
+    <section class="recording-method">
+      <h3>
+        <i class="header-icon"><PlayIcon /></i>Runnable recording
```

Comment:
FYI in the docs this is termed "Code block recording" so we should get consistent on that term. https://appmap.io/docs/recording-methods.html#code-block-recording

EDIT: We actually have 3 different terms for this, because Java calls it Runnable Recording, and Python calls it Context Manager Recording: https://appmap.io/docs/reference/appmap-python.html#context-manager-recording

EDIT: and in Ruby we just call it "Block recording" so we have 4 ways to describe it. I think we should settle on "Code block recording" for now and I'll change the docs site to use just that term.

Reply:
@brikelly before standardizing for consistencies sake, I'd like to understand why the different names were chosen. Does the distinct jargon make sense in the context of each language?

Reply:
It’s named according to the convention for each language. That will feel natural to the user. I don’t think we need to unify that. We are trying to feel language and platform native to the user. Using the terminology of that language and platform is part of how we do that. 

To comment on the instructions you need to put yourself in the shoes of a developer of that language. 

Introducing a term for the purpose of consistency doesn’t help me if I am a developer. I care only about the language I’m using.

Reply:
I'm quite familiar with Java, and calling it Runnable Recording still doesn't make sense to me, even though the feature requires a `Runnable` entry point to record. Consistency aside, it doesn't convey the value or purpose of the feature. It's an implementation detail, but IMO we should be describing things in terms of value to the user so they can decide if they want to use it at all. 

> To comment on the instructions you need to put yourself in the shoes of a developer of that language.

Not every Java programmer is going to be super familiar with `Runnable` unless they are routinely creating threads of their own, which they might not be. Java app servers and frameworks have been trying to shield programmers from doing that kind of stuff for a long time. So if we call it `Runnable Recording` then it will be clear only to a subset of users, but if we call it `code block recording` or `code-driven recording` then 100% of them will understand what it's about.

Reply:
I hear what you’re saying but I don’t think either of the terms code block recording or code driven recording really works either. 

“block” is a Ruby term.

“code driven” isn’t really clear as a definition - isn’t it all code driven?

Reply:
stet